spec/hotel_rooms_specs.rb

```diff
@@ -0,0 +1,57 @@
+require_relative 'spec_helper'
```

Comment:
This filename is wrong.  Check out the `Rakefile` the file name should end with `_spec.rb` not `_specs.rb`

Otherwise good initial set of tests